inventory_management_system_api/routers/v1/catalogue_category.py

```diff
@@ -291,7 +291,7 @@ def partial_update_property(
     catalogue_category_property_service: CatalogueCategoryPropertyServiceDep,
 ) -> CatalogueCategoryPropertySchema:
     logger.info(
-        "Partially updating catalogue category with ID %s's property with ID: %s",
+        "Partially updating catalogue category with ID '%s's property with ID '%s'",
```

Comment:
This is missing an apostrophe after the first ID, but I feel `Partially updating catalogue category with ID '572eaace-3436-4f36-9b0e-cdac5125fe8d''s property with ID '572eaace-3436-4f36-9b0e-cdac5125fe8e'` also doesnt work.

How about rephrasing it to something like
`Partially updating property with ID '572eaace-3436-4f36-9b0e-cdac5125fe8e' within the catalogue category with ID '572eaace-3436-4f36-9b0e-cdac5125fe8d'`, reversing the order to avoid it?